heartbeat/proto/heartbeat.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+package proto;
+
+option go_package = "heartbeat";
+
+// HeartbeatV2 represents the heartbeat message that is sent between peers from the same shard containing
+// current node status
+message HeartbeatV2 {
+  bytes   Payload         = 1;
+  string  VersionNumber   = 2;
+  string  NodeDisplayName = 3;
+  string  Identity        = 4;
+  uint64  Nonce           = 5;
```

Comment:
I think peer subtype is still required to differentiate between regular observer and e.g full history nodeds